### DISCUSSION TITLE
Support for RSS feed for the email aliases

### DISCUSSION THREAD

**Discussion by sumibuzz** [simple-login/app#1849] [Feature Requests]
Add support to read the incoming emails via a RSS feed link, possibly add an optional password/api key to secure the feed??

**Comment by LavaBit5** [comment]
Completely useless and insecure feature.

It is mail aliasing service that NOT stores your emails.

Also RSS cannot be encrypted so it will decrease your security and may expose your sensitive data to hackers.
